pkg/app/piped/platformprovider/kubernetes/applier.go

```diff
@@ -63,6 +63,15 @@ func (a *applier) ApplyManifest(ctx context.Context, manifest Manifest) error {
 	if a.initErr != nil {
 		return a.initErr
 	}
+	if a.input.AutoCreateNamespace == true {
+		if err := a.kubectl.CreateNamespace(
+			ctx,
+			a.platformProvider.KubeConfigPath,
+			a.getNamespaceToRun(manifest.Key),
+		); err != nil {
+			return err
+		}
```

Comment:
This will be called at each manifest applying (ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/executor/kubernetes/kubernetes.go#L236) means that the previously applied manifest creates NS may cause the latter manifest apply to return an error in case of creatingNamespace run. I think we should treat the error here more carefully, like:
- error.Is(NamespaceAlreadyExistErr, err) -> ignore
- other error -> return err
WDYT? 🤔

Reply:
thanks for the feedback ❤️ 
I aware that so rather than call CreateNamespace, for the time being, I was though can have a func CheckNamespaceExist but sound be same because we run it every time manifest applying
so in the CreateNamespace func, it won't throw error if namespace already exist 

https://github.com/pipe-cd/pipecd/pull/4504/files#diff-0f62d61e20f316f805df0111036eaec110ebcbb6792639dda6bb483581832df3R237-R242
just wonder if I can have other better output rather than using `fmt.Printf`

Reply:
Yes, you can check out this implementation 😄 I mean, we can create a new error named `errorNamespaceAlreadyExists` like this one. And at this place, we can check whether the error here is that error or not, if that error is already exist, just ignore it, or else return err.
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/platformprovider/kubernetes/kubectl.go#L31-L32

Reply:
@khanhtc1202 thanks, I don't mind to implement the handle the error if ns already exist which also came in mind when I started 😅, just though if we need it or not because as I mentioned above, in the func CreateNamespace will not throw error if it exist. Please correct me if I missing something

Reply:
Well, nevermind, I think got your point, will Implement that. Thank you 😀 @khanhtc1202

Reply:
@khanhtc1202 again, thank you, I implemented handle err and ignore if match the case